system/jlib/jstring.cpp

```diff
@@ -2482,3 +2501,49 @@ const char * nullText(const char * text)
     if (text) return text;
     return "(null)";
 }
+
+StringBuffer& StringBuffer::operator=(StringBuffer&& value)
+{
+    swapWith(value);
+    return *this;
+}
+
+StringBuffer operator+ (const StringBuffer& lhs, const StringBuffer& rhs)
```

Comment:
I'm not sure about these.  This is introducing a new style of using stringbuffers.  What advantage does it bring?